src/features/payments/shared/components/SendActionList.tsx

```
@@ -0,0 +1,97 @@
+        methods: availableMethods,
+    })
+
+    const handleMethodClick = (method: PaymentMethod) => {
+        // for all methods, save current url and redirect to setup with add-money as final destination
+        // verification will be handled in the add-money flow after login
+        switch (method.id) {
+            case 'bank':
+            case 'mercadopago':
+            case 'pix':
+                saveRedirectUrl()
+                const redirectUri = encodeURIComponent('/add-money')
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)